conans/client/tools/files.py

```diff
@@ -174,6 +174,11 @@ def untargz(filename, destination=".", pattern=None, strip_root=False):
                     name = member.name.replace("\\", "/")
                     member.name = name.split("/", 1)[1]
                     member.path = member.name
+                    if member.linkpath.startswith(common_folder):
```

Comment:
There is another ``untargz`` copy in ``conan.tools.files`` (the one that will survive in 2.0), which should be fixed too? Maybe even the unittest should test it, so it doesn't break when we merge this to ``develop2``?

Reply:
Oh! of course.